modules/rosa-creating-cluster.adoc

```diff
@@ -20,21 +20,25 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 
 .Procedure
 
-. Enter the following command to create your cluster with the default cluster settings.
-+
-[source, terminal]
-----
-$ rosa create cluster --cluster-name=rh-rosa-test-cluster1
-----
+. Create a cluster:
 +
 [NOTE]
 ====
 Multiple availability zones (AZ) are recommended for production workloads. The default is a single availability zone. Use `--help` for an example of how to set this option manually or use interactive mode to be prompted for this setting.
-
-To view other options when creating a cluster, enter `rosa create cluster --help`.
-
-To follow a set of interactive prompts, enter `rosa create cluster --interactive`.
 ====
+
+.. To create your cluster with the default cluster settings:
```

Comment:
Should this be an unordered list, using bullets, rather than ordered, as the user only does one of the sub-steps, not all?

Reply:
Done.